content/200-orm/200-prisma-client/500-deployment/301-edge/485-deploy-to-vercel.mdx

```diff
@@ -0,0 +1,458 @@
+---
+title: 'Deploy to Vercel Edge Functions & Middleware'
```

Comment:
"Middleware" is only mentioned here in the title, but the effective use cases of our users will probably include Middlewares much more often. Can we make sure this is mentioned in more places across the document?

Reply:
added "middleware" to the intro and section heading "General considerations when deploying to Vercel Edge Functions & Edge Middleware" as well